packages/msw/src/requestHandlerRegistry.ts

```diff
@@ -0,0 +1,14 @@
+import type { RestHandler } from "msw";
+
+export class RequestHandlerRegistry {
+    readonly #handlers: RestHandler[] = [];
+
+    add(handlers: RestHandler[]) {
+        this.#handlers.push(...handlers);
+    }
+
+    get handlers(): RestHandler[] {
```

Comment:
unecessary typing : RestHandler[]

Reply:
Sadly it’s necessary otherwise I get that weird TS error at compilation that the type is unreachable. I will add a comment.

Reply:
The `TS2742` error.

Reply:
alright i see.